.claude/commands/TEST_AUTOMATION_STRATEGY.md

```
@@ -0,0 +1,663 @@
+    @DisplayName("{시나리오 설명}")
+    void should{Scenario}() throws Exception {
+        // TODO: Claude Code에게 상세 시나리오 요청
+        // /test-gen-e2e "{시나리오 설명}"
+    }
+}
+```
+
+## Step 2: Claude Code 호출
+→ `/test-gen-e2e "상세 시나리오"`
+→ Claude가 Multi-step 워크플로우 자동 생성
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)